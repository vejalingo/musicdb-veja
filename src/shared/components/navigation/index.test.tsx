import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import Navigation from "./";

describe("Nav", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Router>
          <Navigation />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
