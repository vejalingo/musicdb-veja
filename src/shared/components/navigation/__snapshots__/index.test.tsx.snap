// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<nav
  className="navigation"
>
  <img
    alt="Logo"
    className="navigation__image"
    src="https://lh3.googleusercontent.com/r55K1eQcji3QMHRKERq6zE1-csoh_MTOHiKyHTuTOblhFi_rIz06_8GN5-DHUGJOpn79"
  />
  <input
    type="search"
  />
</nav>
`;
